tests/test_gui_bootstrap_report.py

```
@@ -0,0 +1,95 @@
+RAG bootstrap summary:
+- store_dir: logs/rag_store/demo
+- files_scanned: 2
+- scanned: 5
+- embedded: 3
+- upserted: 3
+- history_records_before: 0
+- history_records_after: 3
+"""
+        summary = summarize_rag_bootstrap_output(output, 0)
+        self.assertEqual(summary.status, "ready")
+        self.assertIn("store_dir：logs/rag_store/demo", summary.facts)
```

Ruff (0.15.17)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)